guides/v2.1/cloud/release-notes/cloud-tools.md

```diff
@@ -18,14 +18,96 @@ You can list the available ece-tools commands using:
+
+-  <!--MAGECLOUD-2491-->Fixed a Sitemap processing issue in the `ece-tools` package that caused third-party extension conflicts after using the `ece-tools` package to apply recent patches that included Sitemap-related changes.
+
+- <!-- MAGECLOUD-2182-->Fixed an issue with the [static content compression process]({{ page.baseurl }}/cloud/env/variables-deploy.html#scd_compression_level) process (`gzip`) that caused `not overwritten` and `no such file or directory` errors when referencing the compressed file during the deployment process.
+
+- <!--MAGECLOUD-2097-->Fixed permission checks that caused `Missing write permissions` errors during the upgrade process.
+
+- <!--MAGECLOUD-2444-->Fixed an issue that prevented the `php ./vendor/bin/ece-tools config:dump` command from removing redundant sections from the `config.php` file during the dump process if the store locale is not specified.
+Now you can easily move your configuration files between environments.
+
+  If you generated `config.php` files using earlier versions of the `ece-tools` package, update to `ece-tools` v2002.0.13 and regenerate the files using the improved command. See [Configuration management for store settings]({{ page.baseurl }}/cloud/live/sens-data-over.html).
+
+- <!--MAGECLOUD-2556-->Fixed an  issue that caused an error during the deploy phase if the route configuration in the `.magento/routes.yaml` file redirects from a an [apex](https://blog.cloudflare.com/zone-apex-naked-domain-root-domain-cname-supp/) domain (also referred to as a non-www or *naked* domain) to a www domain.
+
+-  <!--MAGECLOUD-2520-->Fixed an issue with the `_merge` option for the [`SEARCH_CONFIGURATION`]({{ page.baseurl }}/cloud/env/variables-deploy.html#search_configuration) variable that caused incorrect merge results if the updated `.magento.env.yaml` configuration file did not include the `engine` parameter value along with other updated values. Now, the merge operation correctly overwrites only the parameter values included in the updated `.magento.env.yaml` without requiring you to specify the `engine` parameter value.
+
+-  <!-- MAGECLOUD-2515-->Fixed a Redis configuration issue that incorrectly enabled session locking for {{site.data.var.ece}} v2.1.11+ and v2.2.1+, which can cause slow performance and timeouts. Now, session locking is disabled by default. The issue was caused by a change in the default behavior of the `disable_locking` parameter introduced in v1.3.4 of the Redis session handler package [colinmollenhour/php-redis-session-abstract package](https://github.com/colinmollenhour/php-redis-session-abstract).
+
+- <!--MAGECLOUD-2464-->Fixed an issue in the `ece-tools` package that caused problems with cron queue management in {{site.data.var.ee}} versions 2.2.0 to 2.2.5.
```

Comment:
Should be moved under cron-related fixes but it still doesn't reflect the nature of fix itself:
- In 2.2.0+ we launch consumers with crons by default. On infra's side we have 3 nodes for PRO staging & production. The problem was in incorrect locking mechanism which was supposed to prevent launching same consumers multiple times on HA cluster, since one node was not aware that 2-nd/3-rd was launching same process.
- With the fix all 3 nodes are aware that 1 process of some consumer was already launched by cron and won't be launching same consumer second time.
- Saying this, locking mechanism was fixed for running consumers with crons.